packages/eas-cli/src/devices/utils/formatDevice.ts

```diff
@@ -20,10 +21,34 @@ export default function formatDevice(device: Device, team?: AppleTeamIdAndName):
 
   if (team) {
     fields.push(
-      ...[
-        { label: 'Apple Team ID', value: team.appleTeamIdentifier },
-        { label: 'Apple Team Name', value: team.appleTeamName ?? 'Unknown' },
-      ]
+      { label: 'Apple Team ID', value: team.appleTeamIdentifier },
+      { label: 'Apple Team Name', value: team.appleTeamName ?? 'Unknown' }
+    );
+  }
+
+  return formatFields(fields);
+}
+
+// TODO: integrate this with `formatDevice`
+const DEVICE_CLASS_DISPLAY_NAMES: Record<AppleDeviceClass, string> = {
+  [AppleDeviceClass.Iphone]: 'iPhone',
+  [AppleDeviceClass.Ipad]: 'iPad',
+};
+
+export function formatNewDevice(device: NewDevice, team?: AppleTeamIdAndName): string {
+  const fields = [
```

Comment:
```suggestion
  const fields: { label: string; value: string }[] = [
```
or (preferably) export the type from `formatFields.ts`